src/olympia/addons/migrations/0005_auto_20200527_0513.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.12 on 2020-05-27 05:13
+from django.db import migrations
+
+from olympia.amo.utils import chunked
+
+
+def add_all_addon_guids_to_addon_guid(apps, schema_editor):
+    AddonGUID = apps.get_model('addons', 'AddonGUID')
+    Addon = apps.get_model('addons', 'Addon')
+
+    batch_size = 1000
+    addon_qs = Addon.unfiltered.exclude(guid=None).values('id', 'guid')
+    for addons in chunked(addon_qs, batch_size):
+        addonguids = [
+            AddonGUID(
+                addon_id=addon['id'], guid=addon['guid']) for addon in addons]
+        AddonGUID.objects.bulk_create(addonguids, batch_size)
```

Comment:
@diox for second opinion on this migration

Reply:
This might be a little too much for a migration... I guess we can test that on dev/stage and see if it takes too long...

In prod, we'll likely want the site in read only to ensure that no new add-ons are created before the new code is up - Otherwise there is a window between the guids being fetched and new code being deployed where add-ons would not have a corresponding `AddonGUID`.